agent/eventstream/eventstream.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Package handler deals with appropriately reacting to all ACS messages as well
+// as maintaining the connection to ACS.
+package eventstream
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cihub/seelog"
+	"golang.org/x/net/context"
+)
+
+type eventHandler func(...interface{}) error
```

Comment:
Do you need `...interface{}` here or would `interface{}` be better?  I only see these getting invoked with a single argument.